src/windows/appw.c

```diff
@@ -183,10 +183,10 @@ static void app_register_raw_input(USHORT usage_page, USHORT usage, DWORD flags,
 static void app_adjust_window_rect(MTY_App *app, float scale, RECT *r)
 {
 	if (app->AdjustWindowRectExForDpi) {
-		app->AdjustWindowRectExForDpi(r, WS_OVERLAPPEDWINDOW, FALSE, 0, lrint(scale * 96));
+		app->AdjustWindowRectExForDpi(r, WS_CAPTION, FALSE, 0, lrint(scale * 96));
 
 	} else {
-		AdjustWindowRect(r, WS_OVERLAPPEDWINDOW, FALSE);
+		AdjustWindowRect(r, WS_CAPTION, FALSE);
```

Comment:
Just looked at what WS_CAPTION does compared to WS_OVERLAPPEDWINDOW. It appears to be the case that with this change we're throwing away the min/max buttons as well as the system context menu when you right-click on the title bar. Is this intentional? I don't see how it's relevant to the fix.

Reply:
[Window styles](https://learn.microsoft.com/en-us/windows/win32/winmsg/window-styles)

WS_OVERLAPPEDWINDOW;
- WS_OVERLAPPED (correct, but not allowed for AdjustWindowRect)
- WS_CAPTION (correct and allowed)
- WS_SYSMENU (not necessary for determining size)
- WS_THICKFRAME (We do not use a thick frame)
- WS_MINIMIZEBOX (not necessary for determining size)
- WS_MAXIMIZEBOX (not necessary for determining size)

WS_CAPTION is the minimal set of flags necessary to define the correct window sizing

Reply:
Ah I get it, it's just a query for size, not _actually_ changing the window style.